### PR TITLE
chore: cleanup

### PR DESCRIPTION
**Pull request by CodeWithEmad**
This just cleans the repo readme and some newlines in `.in` files,

- edx -> Open edX
- trailing whitespace removed.
- new lines in `.in` requirements files.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @CodeWithEmad!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.


#### :radio_button: Update the status of your PR

Your PR is currently marked as a draft.  After completing the steps above, update its status by clicking "Ready for Review", or removing "WIP" from the title, as appropriate.




#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?



This repository is currently unmaintained.

To get help with finding a technical reviewer, tag the community contributions project manager for this PR in a comment and let them know that your changes are ready for review:

1. On the right-hand side of the PR, find the Contributions project, click the caret in the top right corner to expand it, and check the "Primary PM" field for the name of your PM.
2. Find their GitHub handle [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3548807177/Community+Contributions+Project+Manager#Current-OSPR-Project-Managers).


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

**Comment by CodeWithEmad** [comment]
My changes are harmless. I don't see why tests should fail.

**Comment by itsjeyd** [comment]
Thank you for this contribution @CodeWithEmad!

File: CHANGELOG.rst
Comment (pdpinch) [inline CHANGELOG.rst:22]: I'm not sure what style guide you're following (if any), but it seems like you added code formatting to file names in other places. Do you want to add it here?

i.e. 
```suggestion
- Removed ``openedx.yaml`` file
```

**Review by pdpinch — APPROVED** [review]
Looks good to me. 

I made one small comment, which you can take or leave.

File: CHANGELOG.rst
Comment (CodeWithEmad) [reply CHANGELOG.rst:22]: Nice catch, @pdpinch. I'll fix it.

Comment (CodeWithEmad) [reply CHANGELOG.rst:22]: Nice catch, @pdpinch. I'll fix it.

**Comment by itsjeyd** [comment]
@pdpinch Thanks for jumping in and providing a review for this PR!

Would you be able to merge the changes or will we need to ask for help from Axim for that?

@CodeWithEmad Could you please look into why some of the automated tests are failing?

**Comment by CodeWithEmad** [comment]
See https://github.com/openedx/edx-cookiecutters/pull/475#issuecomment-2368047756

**Comment by itsjeyd** [comment]
See https://github.com/openedx/edx-cookiecutters/pull/475#issuecomment-2377049784

**Comment by itsjeyd** [comment]
[Same here](https://github.com/openedx/edx-cookiecutters/pull/475#issuecomment-2406971157), marking as blocked on #471.

**Comment by itsjeyd** [comment]
@CodeWithEmad Now that #471 has been merged I'm removing the blocker label. Back to you for now. Once the build is green we should be good to merge this.

**Comment by CodeWithEmad** [comment]
@itsjeyd Done

**Comment by itsjeyd** [comment]
@CodeWithEmad Great!

@feanil Since this repo is unmaintained could you please do the honors and merge the changes? They were approved via https://github.com/openedx/edx-cookiecutters/pull/474#pullrequestreview-2290304211.

**Review by feanil — APPROVED** [review]

**Pull request merged**
